src/components/App.jsx

```diff
@@ -9,7 +13,11 @@ export const App = () => {
         color: '#010101',
       }}
     >
-      React homework template
+      <Routes>
+        <Route path="/soyummy_frontend" element={<StartPage />} />
+        <Route path="/signin" />
+        <Route path="/registration" />
```

Comment:
If you implement something later on please comment that line for future.
In this code we are missing `element` for both path's. ❌

Reply:
Fixed 👍